src/QuickMarcEditor/QuickMarcEditWrapper.test.js

```diff
@@ -439,6 +466,38 @@ describe('Given QuickMarcEditWrapper', () => {
           });
         });
       });
+
+      it('should actualize new linked fields', async () => {
+        const { getByText } = renderQuickMarcEditWrapper({
+          instance,
+          mutator,
+        });
+
+        await act(async () => { fireEvent.click(getByText('stripes-acq-components.FormFooter.save')); });
+
+        const formValues = mockFormValues(MARC_TYPES.BIB);
+
+        const prepareForSubmit = (values) => {
+          return flow(
+            removeDeletedRecords,
+            removeDuplicateSystemGeneratedFields,
+          )(values);
+        };
+
+        const formValuesToProcess = flow(
```

Comment:
This test is tied too much to implementation. It's better to define two constants (initial values and expected values), mock click on save and make sure actualize links was called with expected values